draft-ietf-quic-recovery.md

```diff
@@ -880,9 +880,10 @@ similar to a sender's response on a Retransmission Timeout (RTO) in TCP
 
 This document does not specify a pacer, but it is RECOMMENDED that a sender pace
 sending of all in-flight packets based on input from the congestion
-controller. For example, a pacer might distribute the congestion window over
-the smoothed RTT when used with a window-based controller, or a pacer might use
-the rate estimate of a rate-based controller.
```

Comment:
I'm OK with leaving this out, but I'm interested in why you did.

Reply:
I thought it was redundant to the text you added below.